core/store/tx_manager.go

```diff
@@ -241,8 +247,16 @@ func (txm *EthTxManager) createTx(
 		}
 
 		logger.Warnw(
-			"Tx #0: nonce too low, retrying with network nonce",
-			"nonce", tx.Nonce, "error", err.Error(),
+			"Tx #0: another tx with this nonce already exists, will retry with network nonce",
+			"nonce", tx.Nonce, "gasPriceWei", gasPriceWei, "gasLimit", gasLimit, "error", err.Error(),
+		)
+
+		// Linear backoff
+		time.Sleep(time.Duration(nrc+1) * nonceReloadBackoffBaseTime)
```

Comment:
Actually, this should be 
`		time.Sleep(time.Duration(nrc) * nonceReloadBackoffBaseTime)` No need to wait on the happy path.

Reply:
This is not the happy path. This implies one failure already.